src/universal-dapp.js

```diff
@@ -53,30 +53,6 @@ UniversalDApp.prototype.getAccounts = function (cb) {
     }
 };
 
-UniversalDApp.prototype.getBalance = function (address, cb) {
```

Comment:
Cool. Is anyone working on that feature?

On Thursday, June 2, 2016, Alex Beregszaszi <notifications@github.com>
wrote:

> In src/universal-dapp.js
> <https://github.com/ethereum/browser-solidity/pull/49#discussion_r65597491>
> :
>
> > @@ -53,30 +53,6 @@ UniversalDApp.prototype.getAccounts = function (cb) {
> >      }
> >  };
> >
> > -UniversalDApp.prototype.getBalance = function (address, cb) {
>
> This was specifically added for a new feature - to display account
> balances after each transaction.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ethereum/browser-solidity/pull/49/files/acaebcccf4266a148066627c8e6587427305c8a7#r65597491>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAJ0ByHT_nHfJkUVvlZSMqJPUuMjWIaks5qHya4gaJpZM4Is2IN>
> .
>